### ISSUE TITLE
Loss curve

### ISSUE THREAD

**Issue by Kevin43614** [bubbliiiing/centernet-pytorch#37]
您好 我想請問一下訓練時 training loss 總是大於 validation loss 是有可能的嗎?
